scripts/examples/python/setup.cfg

```diff
@@ -0,0 +1,27 @@
+[metadata]
+name = XenAPI
+version = attr: src.VERSION
+description = XenAPI SDK, for communication with Citrix Hypervisor.
+author = Citrix Systems, Inc.
+author_email = xen-api@lists.xenproject.org
+url = http://community.citrix.com/display/xs/Download+SDKs
+project_urls =
+    Bug Tracker = https://github.com/xapi-project/xen-api/issues,
```

Comment:
This comma ends up in the URL, which then causes it to 404, can you drop the comma?

Reply:
let me see, the manual says the entries in a dict are comma-separated